committee-steering/governance/wg-governance.md

```diff
@@ -0,0 +1,103 @@
+
+Therefore, Working Group formation begins with the organizers asking themselves some important questions that
+should eventually be reflected in a pull request on sigs.yaml:
+
+1. What is the exact problem this group is trying to solve?
+1. What is the artifact that this group will deliver, and to whom?
+1. How does the group know when the problem solving process is completed, and it is time for the Working Group to
+   dissolve?
+1. Who are all of the stakeholders involved in this problem this group is trying to solve (SIGs, steering committee,
+   other Working Groups)?
+1. What are the meeting mechanics (frequency, duration, roles)?
+1. Does the goal of the Working Group represent the needs of the project as a whole, or is it focused on the interests
+   of a narrow set of contributors or companies?
+1. Who will chair the group, and ensure it continues to meet these requirements?
+1. Is diversity well-represented in the Working Group?
+
+Once the above questions have been answered, the pull request against sigs.yaml can be created. Once the generator
+is run, this will in turn create the OWNERS_ALIASES file, readme files, and the main SIGs list.  The minimum
+requirements for that are:
+
+- name
+- directory
+- mission statement
+- chair information
+- meeting information
+- contact methods
+- any sig stakeholders
+- any subproject stakeholders
+
+The pull request should be labeled with any SIG stakeholders and committee/steering. And since GitHub notifications
+are not a reliable means to contact people, an email should be sent to the mailing lists for the stakeholder SIGs,
+and the steering committee with a link to the PR. A member of the community admin team will place a /hold on it
+until it has an LGTM from at least one chair from each of the stakeholder SIGs, and a simple majority of the steering
+committee.
+
+Once merged, the Working Group is officially chartered until it either completes its stated goal, or disbands
+voluntarily (e.g. due to new facts, member attrition, change in direction, etc). Working groups should strive to
+make regular reports to stakeholder SIGs in order to ensure the mission is still aligned with the current state.
+
+Deliverables (documents, diagrams) for the group should be stored in the directory created for the Working Group.
+If the deliverable is a KEP, it would be helpful to link it in the closed formation/charter PR for future reference.
+
+## Disbandment Process Description
+
+Working Groups will be disbanded if either of the following is true:
+
+- There is no long a Chair
+  - (with a 4 week grace period)
+- None of the communication channels for the Working Group have been in use for the goals outlined at the founding of
+  the Working Group
+  - (with a 3 month grace period)
+  - Slack
+  - Email Discussion Forum
+  - Zoom
+
+The current Chair may step down at any time.  When they do so, a new Chair may be selected through lazy consensus
+within the Working Group.
```

Comment:
and relevant section in sigs.yaml should be updated to reflect that. right?

Reply:
#2814